pasta_eln/GUI/tableHeader.py

```diff
@@ -68,7 +68,7 @@ def execute(self, command:list[Any]) -> None:
     if command[0] is Command.ADD:
       self.selectedList += selectedLeft
     elif command[0] is Command.DELETE:
-      self.selectedList = [i for i in self.selectedList if i not in selectedRight ]
+      self.selectedList = [i for i in self.selectedList if i not in selectedRight or i in ['name'] ]
```

Comment:
if it's just name, why can't we replace the statement 'i in ['name']' with  "i == 'name'" ?